aloof/aloof.py

```diff
@@ -5,7 +5,7 @@
 """
 
 import sys
-import subprocess 
+import subprocess
```

Comment:
Doesn't look like it to me. Seems to run without those lines. It was in the original version by @kunzel I just shuffled it around a bit by accident.